src/python/WMCore/WMRuntime/Scripts/SetupCMSSWPset.py

```diff
@@ -434,14 +434,13 @@ def _processPileupMixingModules(self, pileupDict, seLocalName, modules, requeste
                     if requestedPileupType == 'data':
                         baggage = self.job.getBaggage()
                         if getattr(baggage, 'skipPileupEvents', None) is not None:
+                            # For deterministic pileup, we want to shuffle the list the
+                            # same for every job in the task and skip events
+                            random.seed(self.job['task'])
```

Comment:
@ericvaandering here is the print baggage.skipPileupEvents location I was talking about (which gets printed in scramOutput.log, for future reference...).
Once you add this print statement, @ticoann  will merge it and cut a new release.